recipe/make.inc

```diff
@@ -0,0 +1,3 @@
+BACKEND = cuda
+FORT = false
+GPU_TARGET=sm_35
```

Comment:
???

Reply:
This file is required for the code generation step.